scripts/zkai_d128_down_projection_proof_input.py

```
@@ -0,0 +1,719 @@
+    "down_projection_mul_row_commitment",
+    "residual_delta_relabels_full_output",
+    "non_claims",
+    "next_backend_step",
+)
+
+
+class D128DownProjectionInputError(ValueError):
+    pass
+
+
+def _load_module(path: pathlib.Path, module_name: str) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_load_module`

(ANN401)